packages/fxa-settings/src/pages/ResetPassword/AccountRecoveryResetPassword/index.tsx

```diff
@@ -64,35 +107,17 @@ const AccountRecoveryResetPassword = ({
       },
     });
 
-  const alertSuccessAndNavigate = useCallback(() => {
-    const successCompletePwdReset = ftlMsgResolver.getMsg(
-      `${viewName}-success-alert`,
-      'Password set'
-    );
-    alertBar.success(successCompletePwdReset);
-    navigate('/reset_password_with_recovery_key_verified', { replace: true });
-  }, [alertBar, ftlMsgResolver, navigate]);
-
-  const sendNewLinkEmail = () => {
-    // TODO: Hook up functionality to send a new verification link
-  };
-
-  const onSubmit = () => {
-    try {
-      // TODO: - completeAccountPasswordResetWithRecoveryKey
-      //       - logViewEvent('verification.success');
-      //       - metrics.logUserPreferences('account-recovery', false)
-      //       - logFlowEventOnce('recovery-key-consume.success', viewName);
-      alertSuccessAndNavigate();
-    } catch (e) {
-      // TODO: - if token expired, re-render and show LinkExpired
-      //       - metrics event for error
-      //       - error feedback in alertBar
-    }
-  };
+  // Make some presentation decisions based on initial state
+  if (state.contextError) {
+    alertInvalidContext(state.contextError);
+  } else if (!state.supportsRecovery) {
+    alertBar.error('Redirecting');
```

Comment:
Q: Is this user facing, and does it need to be localized?

Reply:
I am not sure. I am also not sure it's necessary. The redirect should happen almost immediately. In the even it hangs, or is slow for whatever reason, it might be nice to show an alert. It probably shouldn't be an error, and probably should translated. I'd love to get more opinions on this though. It might simply go away too.

Reply:
@vbudhram I could use another opinion on whether or not this is necessary. It was 'nice' to see in storybook. That's why I added it...